Doc/library/sqlite3.rst

```diff
@@ -1111,15 +1136,13 @@ Row Objects
 
 .. class:: Row
 
-   A :class:`Row` instance serves as a highly optimized
+   A :class:`Row` instance that serves as a highly optimized
    :attr:`~Connection.row_factory` for :class:`Connection` objects.
-   It tries to mimic a tuple in most of its features.
-
-   It supports mapping access by column name and index, iteration,
+   It tries to mimic a :class:`tuple` in most of its features,
+   and supports :term:`mapping` access by column name and index, iteration,
    representation, equality testing and :func:`len`.
 
-   If two :class:`Row` objects have exactly the same columns and their
-   members are equal, they compare equal.
+   Two row objects are equal if they have equal columns and equal members.
```

Comment:
```suggestion
   Two row objects compare equal if have equal columns and equal members.
```

* Use slightly clearer and more idiomatic phrasing

Reply:
No _they_? TIL